core/src/test/java/org/apache/iceberg/ScanPlanningAndReportingTestBase.java

```diff
@@ -47,7 +47,7 @@ public abstract class ScanPlanningAndReportingTestBase<
 
   @Parameters(name = "formatVersion = {0}")
   public static List<Object> parameters() {
-    return Arrays.asList(2, 3);
+    return Arrays.asList(2, 3, 4);
```

Comment:
nit: Maybe we should change this to use the MAX_VERSION or all Versions parameter with a (not 1) or something